app/controllers/api/v2/hosts_controller.rb

```diff
@@ -197,6 +197,27 @@ def get_status
         end
       end
 
+      api :DELETE, "/hosts/:id/status/:type", N_("Clear sub-status of host")
+      param :id, :identifier_dottable, :required => true
+      param :type, [ HostStatus::Global ] + HostStatus.status_registry.to_a.map { |s| s.humanized_name }, :required => true, :desc => N_(
+        <<~EOS
+          status type
+        EOS
+      )
+
+      description N_('Clears a host sub-status of a given type')
+      def forget_status
+        status = @host.get_status(HostStatus.find_status_by_humanized_name(params[:type]))
+        if params[:type].eql?('global')
```

Comment:
Nit: In ruby you tyically use `==` for comparisons.
```suggestion
        if params[:type] == 'global'
```